src/Files.App/Views/Layouts/ColumnLayoutPage.xaml

```diff
@@ -13,137 +12,19 @@
-								ContentTransitions="{TemplateBinding ContentTransitions}" />
-
-							<Rectangle
-								Grid.Column="1"
-								Height="1"
-								HorizontalAlignment="Stretch"
-								VerticalAlignment="Center"
-								Stroke="{ThemeResource GridViewHeaderItemDividerStroke}"
-								StrokeThickness="0.5" />
-
-							<VisualStateManager.VisualStateGroups>
-								<VisualStateGroup x:Name="CommonStates">
-									<VisualState x:Name="Normal" />
-									<VisualState x:Name="PointerOver">
-										<Storyboard>
-											<ObjectAnimationUsingKeyFrames Storyboard.TargetName="HeaderItemRootGrid" Storyboard.TargetProperty="Background">
-												<DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBackgroundPointerOver}" />
-											</ObjectAnimationUsingKeyFrames>
-											<ObjectAnimationUsingKeyFrames Storyboard.TargetName="HeaderItemRootGrid" Storyboard.TargetProperty="BorderBrush">
-												<DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBorderBrushPointerOver}" />
-											</ObjectAnimationUsingKeyFrames>
-										</Storyboard>
-										<VisualState.Setters>
-											<!--<Setter Target="ContentPresenter.(local:AnimatedIcon.State)" Value="PointerOver" />-->
-										</VisualState.Setters>
-									</VisualState>
-									<VisualState x:Name="Pressed">
-										<Storyboard>
-											<ObjectAnimationUsingKeyFrames Storyboard.TargetName="HeaderItemRootGrid" Storyboard.TargetProperty="Background">
-												<DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBackgroundPressed}" />
-											</ObjectAnimationUsingKeyFrames>
-											<ObjectAnimationUsingKeyFrames Storyboard.TargetName="HeaderItemRootGrid" Storyboard.TargetProperty="BorderBrush">
-												<DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonBorderBrushPressed}" />
-											</ObjectAnimationUsingKeyFrames>
-										</Storyboard>
-										<VisualState.Setters>
-											<!--<Setter Target="ContentPresenter.(local:AnimatedIcon.State)" Value="Pressed" />-->
-										</VisualState.Setters>
-									</VisualState>
-								</VisualStateGroup>
-							</VisualStateManager.VisualStateGroups>
-						</Grid>
-					</ControlTemplate>
-				</Setter.Value>
-			</Setter>
-		</Style>
-
-		<Style
-			x:Name="CompactItemContainerStyle"
-			BasedOn="{StaticResource DefaultListViewItemStyle}"
-			TargetType="ListViewItem">
-			<Setter Property="HorizontalContentAlignment" Value="Stretch" />
-			<Setter Property="MinHeight" Value="1" />
-			<Setter Property="Margin" Value="-2" />
-			<Setter Property="Padding" Value="4,0" />
-		</Style>
-
-		<Style
-			x:Name="RegularItemContainerStyle"
-			BasedOn="{StaticResource DefaultListViewItemStyle}"
-			TargetType="ListViewItem">
-			<Setter Property="HorizontalContentAlignment" Value="Stretch" />
-			<Setter Property="MinHeight" Value="1" />
-			<Setter Property="Margin" Value="0" />
-			<Setter Property="Padding" Value="4,0" />
-		</Style>
```

Comment:
These item styles are only needed in this view, isn't it better to keep them here to avoid unnecessary resources in other views?